lib/util.js

```diff
@@ -573,7 +585,8 @@ exports.isPrimitive = isPrimitive;
 function isBuffer(arg) {
   return arg instanceof Buffer;
 }
-exports.isBuffer = isBuffer;
+exports.isBuffer = deprecate(isBuffer,
+    'util.isBuffer is deprecated, please use a user-land alternative.');
```

Comment:
OK, should I use `instanceof Buffer` instead of  `util.isBuffer` in Node `0.12.7` and above?